doc/release_history.rst

```diff
@@ -1,23 +1,295 @@
-
+###############
```

Comment:
@mtreinish i know this is strange my convention but is because when i make the tutorials it has the same heading

Reply:
it was throwing a warning in sphinx because it was getting confused by the levels (see Diego's earlier comment). I just referenced the sphinx documentation: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections to make sure sphinx didn't get confused